src/utils/local_path.ts

```diff
@@ -45,3 +45,34 @@ export function getOutputFilePathAndName(
 	}
 	throw new Error(`The destination is neither a folder nor a file!`);
 }
+
+/**
+ * Resolves the path, verifies its existance and type to conditionally return its dir and file name
+ * @param {string} destOutputPath - the path from where to extract the dir path and name
```

Comment:
params are out of date

Reply:
a38b168c20b24f2c68c42faee952513d1f34a81e